NOTICE.md

```diff
@@ -494,13 +604,65 @@ SLF4J API (1.7.2)
 
 * License: MIT License
 
+slf4j-api (1.7.30)
+
+* License: MIT
+
+slf4j-api (2.0.0)
+
+* License: MIT
+
 spring-agent (2.5.3)
 
 * License: Apache License, 2.0
 * Project: http://www.springframework.org
 * Source:
    http://downloads.sourceforge.net/springframework/spring-framework-2.5.3-with-dependencies.zip?modtime=1204283106&big_mirror=0
 
+spring-aop (5.2.0)
```

Comment:
I think, that all spring components are under Apache-2.0 license.

Reply:
do you see Apache-2.0 license missing somewhere? Some pieces are dual-licensed